src/test/visit_test_main.py

```diff
@@ -967,6 +973,10 @@ def Test(case_name, altSWA=0, alreadySaved=0):
     dpix      = 0.0
     davg      = 0.0
     thrErr    = -1.0
+    if not pixdiff:
```

Comment:
Best to check for `None` vs boolean, b/c this will also trigger if pixdiff is `0`
Example:
```
a = None
b = 0

def doit(v):
    if not v:
        print("!")

def doit2(v):
    if v is None:
        print("!")

print("Check with Not")

doit(a)
doit(b)

print("Check for None")

doit2(a)
doit2(b)
```


Output:
```
Check with Not
!
!
Check for None
!
```

Reply:
Ok, thanks for catching that. Fixed now.